test/common.js

```diff
@@ -141,6 +141,10 @@ Object.defineProperty(exports, 'hasCrypto', {get: function() {
   return process.versions.openssl ? true : false;
 }});
 
+Object.defineProperty(exports, 'hasFipsCrypto', {get: function() {
```

Comment:
Sure, I'll change the line above it too as mine is an exact copy of the existing hasCrypto above it.